Main.java

```diff
@@ -1,15 +1,31 @@
-/**
- *
- * @author 
- */
-public class Main {
-
-  /**
-   * The method that is executed when you hit the run button.
-   * @param args the command line arguments
-   */
-  public static void main(String[] args) {
-    // TODO code application logic here
-    
-  }
-}
+import java.util.Scanner;
+/**
+ * Determines if a given number is odd or even
+ * @jacoe0850 
```

Comment:
So the @author is actually a special tag, something called javadoc. So it should really say @author jacoe0850